Gpt checker (#18)
greenelab/word_lapse_manuscript: content/02.introduction.md

# Introduction

The meaning of words is constantly evolving.
For instance, the word "nice" used to mean foolish or innocent in the 15th-17th centuries, before it underwent a shift to its modern meaning of "pleasant or delightful" [@doi:10.1093/acrefore/9780199384655.013.323].
This change can be attributed to writers using new metaphors or substituting words with similar meanings, a process known as metonymy [@doi:10.1093/acrefore/9780199384655.013.323].
By studying these shifts, we can gain a more nuanced understanding of how language adapts to describe our world.

Scientific fields of inquiry are constantly evolving as researchers develop and test new hypotheses and applications.

> For example, the CRISPR-Cas9 system has been repurposed as a tool for genome editing, transforming how we discuss molecular entities.

Reviewer: ```suggestion
For example, in the interval studied the CRISPR-Cas9 system has been repurposed as a tool for genome editing.
```

Other patterns include that words change meaning inversely proportional to their frequency, and words with multiple meanings have higher rates of change [@arxiv:1605.09096].
Most of these discoveries have been made in regular English text.
However, researchers have also attempted to investigate whether these patterns are also found in biomedical literature [@doi:10.1016/j.ijmedinf.2017.11.006].
The only strong evidence they found is that words that change meaning do so inversely proportional to their usage frequency [@doi:10.18653/v1/W19-5037].
Despite conflicting evidence, it is clear that biomedical words and concepts change over time.

Recent studies have investigated semantic shifts in various non-biomedical corpora, such as newspapers [@doi:10.18653/v1/W17-2705; @arxiv:1711.05603; @doi:10.1017/S0003055417000570], books [@arxiv:1605.09096], Reddit [@doi:10.1016/j.jbi.2021.103824], and Twitter [@arxiv:1411.3315].
Other research has focused on semantic shifts in topics related to information retrieval [@doi:10.1007/s11192-018-2843-2], and the COVID-19 pandemic has been studied multiple times [@doi:10.1142/9789811232701_0011;@arxiv:2102.07836; @doi:10.2196/22635].

> Additionally, researchers have examined how the usage of drugs and diseases change over time [@doi:10.18653/v1/W19-5037].

Reviewer: ```suggestion
Additionally, researchers have examined how term usage related to drugs and diseases changes over time [@doi:10.18653/v1/W19-5037].
```

> However, with the dramatic increase in the amount of open-access biomedical literature over the last two decades, there is an opportunity to analyze semantic shifts in biomedicine on a global scale.

Reviewer: ```suggestion
However, with the dramatic increase in open-access biomedical literature over the last two decades, there is an opportunity to analyze semantic shifts in biomedicine on a whole-literature scale.
```
